forge/Installation and Upgrading/installubuntuapachemysql.md

```diff
@@ -22,71 +22,43 @@ This installation guide focuses on installing the TAO platform on Ubuntu using A
 
 The first step of the installation of our environment is to install the Apache2 web server (recommended version 2.4). To do so, open up a terminal and type the following command lines. The *apt-get update* command aims at updating your system. You can skip this command if you think your system is up to date.
```

Comment:
What version of Apache did you get on your Ubuntu 16.04? Is it still 2.4? If not, please update the recommended version portion with the version you got.